Makefile

```diff
@@ -110,6 +110,33 @@ deploy-full-local-setup: ## Deploy full local multicluster setup
 	$(call deploy-local-cluster,$(CLUSTER_GSLB1),$(CLUSTER_GSLB2),$(VERSION),)
 	$(call deploy-local-cluster,$(CLUSTER_GSLB2),$(CLUSTER_GSLB1),$(VERSION),$(CLUSTER_GSLB2_HELM_ARGS))
 
+.PHONY: deploy-stable
+deploy-stable:
+	$(call deploy-local-cluster,$(CLUSTER_GSLB1),$(CLUSTER_GSLB2),$(VERSION),)
+	$(call deploy-local-cluster,$(CLUSTER_GSLB2),$(CLUSTER_GSLB1),$(VERSION),$(CLUSTER_GSLB2_HELM_ARGS))
+
+	$(call list-running-pods,$(CLUSTER_GSLB1))
+	$(call list-running-pods,$(CLUSTER_GSLB2))
+
+.PHONY: deploy-candidate
+deploy-candidate:
+	@echo "\n$(YELLOW)build k8gb docker and push to registry $(NC)"
+	docker build . -t $(REPO):$(SEMVER)
+
+	k3d image import $(REPO):$(SEMVER) -c $(CLUSTER_GSLB1)
+	k3d image import $(REPO):$(SEMVER) -c $(CLUSTER_GSLB2)
+
+	@echo "\n$(YELLOW)Bump GSLB operator from $(VERSION) to $(SEMVER) on k3d-$(CLUSTER_GSLB1) $(NC)"
+	kubectl config use-context k3d-$(CLUSTER_GSLB1)
+	$(call deploy-k8gb-helm,$(CLUSTER_GSLB1),$(CLUSTER_GSLB2),$(SEMVER),)
+
+	@echo "\n$(YELLOW)Bump GSLB operator from $(VERSION) to $(SEMVER) on k3d-$(CLUSTER_GSLB2) $(NC)"
+	kubectl config use-context k3d-$(CLUSTER_GSLB2)
+	$(call deploy-k8gb-helm,$(CLUSTER_GSLB2),$(CLUSTER_GSLB1),$(SEMVER),$(CLUSTER_GSLB2_HELM_ARGS))
+
+	$(call list-running-pods,$(CLUSTER_GSLB1))
+	$(call list-running-pods,$(CLUSTER_GSLB2))
+
 # triggered by terraform GitHub Action. Clusters already exists. GO is not installed yet
 .PHONY: deploy-to-AbsaOSS-k3d-action
```

Comment:
This target is not touched by this PR, but just general question: aren't `deploy-stable` and `deploy-candidate` triggered by GHA and deploying to k3d clusters as well? Maybe this target should be renamed as well?

Reply:
@somaritane  I renamed to `deploy-full-candidate`, based on pattern `deploy-full-local-setup`

Reply:
@kuritka , my understanding is that by "full" we mean two-clusters setup. But `deploy-stable` and `deploy-candidate` are also deploying to both clusters, just methods and target versions are different.
In that sense, my suggestion is to rename `deploy-candidate` to `deploy-candidate-with-helm`, and `deploy-full-candidate` to just `deploy-candidate`. What do you think?